src/solidity-frontend/solidity_convert.cpp

```diff
@@ -3458,6 +3464,12 @@ bool solidity_convertert::get_statement(
 
     break;
   }
+  case SolidityGrammar::StatementT::PlaceholderStatement:
+  {
+    // TODO: splite the modifierdefinition node, record content before and after, and prepare to insert
```

Comment:
I think we might need to construct and populate the global auxiliary block to store the modifier.

We need to consider the execute order. Also the inheritance